jscatter/widget.py

```diff
@@ -185,6 +196,20 @@ class JupyterScatter(anywidget.AnyWidget):
     # For synchronyzing view changes across scatter plot instances
     view_sync = Unicode(None, allow_none=True).tag(sync=True)
 
+    def __init__(self, data, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.data = data
+        self.on_msg(self._handle_custom_msg)
```

Comment:
Is there a reason why we are assigning `data` after the `super()` now? As in, whey not:

```python
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.on_msg(self._handle_custom_msg)
```

Reply:
Mhhh, I thought we have to do this since `data` isn't a widget property but we need access to the dataframe in `_handle_custom_msg`

Reply:
Ah interesting. I saw that `jscatter.py`initializes the widget with:

```python
self._widget = JupyterScatter(data=self._data, ...)
```

But I guess we were just doing that before without `data` ever being set as a property.